pyvrp/cpp/Individual.cpp

```diff
@@ -118,15 +118,12 @@ bool Individual::operator==(Individual const &other) const
 {
     // First compare costs, since that's a quick and cheap check. Only when
     // the costs are the same do we test if the neighbours are all equal.
-    return cost() == other.cost() && neighbours == other.neighbours;
+    return distance == other.distance && capacityExcess == other.capacityExcess
+           && timeWarp == other.timeWarp && neighbours == other.neighbours;
```

Comment:
Should this also test the `nbRoutes` item? In terms of how, something like this requires less scanning, I think:
```cpp
return distance == ...
    && capacityExces = ...
    && timeWarp == ...
    && neighbours = ...;
```
This is something I explicitly disable `clang-format` for (with `// clang-format off` and `on`) since the default doesn't really make a lot of sense to me.

Reply:
Yes! It's a bit weird as `numRoutes_` has a `_` whereas the other properties don't but I think that is a separate issue.

Reply:
That's because it's data but also available as a method, and they cannot be named the same. I suspect we'll want to ahve the same for e.g. distance at some point.

Reply:
The comment above this is out of date now too. Can we update that to something along the lines of "first check cheap attributes, and if those all agree we check the expensive thing to really be sure"?

Reply:
> That's because it's data but also available as a method, and they cannot be named the same. I suspect we'll want to ahve the same for e.g. distance at some point.

I will fix this in #191 

> The comment above this is out of date now too. Can we update that to something along the lines of "first check cheap attributes, and if those all agree we check the expensive thing to really be sure"?

Yes, thanks!